examples/custom-esbuild/sanity-esbuild-app-esm/src/app/app.component.spec.ts

```diff
@@ -1,23 +1,46 @@
-import { TestBed, waitForAsync } from '@angular/core/testing';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { AppComponent } from './app.component';
 
 describe('AppComponent', () => {
-  beforeEach(waitForAsync(() => {
```

Comment:
Had to get rid of `waitForAsync` as these "fake async" helpers aren't supported in the Vitest env: https://github.com/angular/angular-cli/issues/31094#issuecomment-3260039353